runconfig/config.go

```diff
@@ -59,6 +60,10 @@ type Command struct {
 	parts []string
 }
 
+func (e *Command) ToString() string {
```

Comment:
Instead of calling this `ToString`, you can just call it `String`, then go will automatically call it.
For example, http://play.golang.org/p/_r7AImK0fi